README.md

```diff
@@ -1,4 +1,37 @@
 # rc-spanishIdValidator [Work in progress] 
 
-This is and AngularJS directive which provides validation for common Spanish official ID numbers like DNI or NIF.
+This is and AngularJS library which provides validation directives for common Spanish official ID numbers like DNI or NIF.
 
+###Usage and examples
+
+The usage of the validation directives is very easy.
```

Comment:
You have to be assertive, please do not make marketing commitments like "is very easy"